extensions/amp-link-rewriter/0.1/test/test-helpers.js

```diff
@@ -27,6 +27,23 @@ const helpersMaker = () => {
       element.append(script);
       return element;
     },
+    createConfig(config = {}) {
```

Comment:
Please remove the `'scopeDocument': false,`, and instead set it in your scope tests. 

Leaving it as is might be confusing to a new contributor as to why `scopeDocument` is defaulting to `false` for a generic `createConfig` method, plus the change will help to make your test more readable.